python/torch_mlir/extras/fx_importer.py

```diff
@@ -136,7 +137,7 @@
     torch.int16: np.int16,
     torch.int32: np.int32,
     torch.int64: np.int64,
-    # torch.bf16: None, there's no equivalent np datatype so this isn't supported right now
+    torch.bfloat16: mld.bfloat16,# , there's no equivalent np datatype so this isn't supported right now
```

Comment:
What do you think about making this a "soft" dependency? Also, let's just spell out ml_dtypes. The abbreviation isn't paying for itself.

I'd recommend at the top:

```
try:
  import ml_dtypes
except ModuleNotFoundError:
  # The third-party ml_dtypes package provides some optional
  # low precision data-types. If used in this file, it is
  # conditional.
  ml_dtypes = None
```

Then instead of doing this inline:

```
if ml_dtypes is not None:
  TORCH_DTYPE_TO_NPY_TYPE[torch.bfloat16] = ml_dtypes.bfloat16
```

Reply:
sounds good